Wrappers/Python/cil/plugins/tigre/ProjectionOperator.py

```diff
@@ -64,22 +70,43 @@ def __init__(self, image_geometry, aquisition_geometry, direct_method='interpola
 
         self.method = {'direct':direct_method,'adjoint':adjoint_weights}
 
+        #set up TIGRE geometry
+        tigre_geom, tigre_angles= CIL2TIGREGeometry.getTIGREGeometry(image_geometry,aquisition_geometry)
+
         #TIGRE bug workaround, when voxelgrid and panel are aligned ray tracing fails
```

Comment:
Is there an issue on TIGRE? Probably not but would be worth opening and referencing